windows/ReactTestApp/ReactInstance.cpp

```diff
@@ -0,0 +1,50 @@
+#include "pch.h"
+#include "ReactInstance.h"
+#include <filesystem>
+
+namespace winrt::ReactTestApp::implementation
+{
+    void ReactInstance::LoadJSBundleFrom(JSBundleSource source)
+    {
+        m_reactNativeHost.InstanceSettings().UseLiveReload(source == JSBundleSource::DevServer);
```

Comment:
Nit: We can create a reference to `ReactInstanceSettings` so we don't have to get it on every line:
```suggestion
        auto &instanceSettings = m_reactNativeHost.InstanceSettings();
        instanceSettings.UseLiveReload(source == JSBundleSource::DevServer);
```